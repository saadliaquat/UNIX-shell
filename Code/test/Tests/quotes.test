find .. -type f -name "*.c" -o -name "*.h" | xargs wc -l | sort -n && find .. -type f -name "*.c" -o -name "*.h" | xargs wc -l | tail -n 1 | awk '{print $1}' | xargs echo "LOC: " ; find .. -type f -name "*.c" -o -name "*.h" > find.log && cat find.log | xargs wc -l | awk '{print $1}' | sort -n | paste -sd+ | bc
find .. -type f -name "*.c" -o -name "*.h" | xargs wc -l | sort -n
find .. -type f -name "*.c" -o -name "*.h" | xargs wc -l | tail -n 1 | awk '{print $1}' | xargs echo "LOC: "
find .. -type f -name "*.c" -o -name "*.h" > find.log && cat find.log | xargs wc -l | awk '{print $1}' | sort -n | paste -sd+
find .. -type f -name "*.c" -o -name "*.h"
alias findCfiles "echo ../src/*.c"
echo "*.c" | grep c