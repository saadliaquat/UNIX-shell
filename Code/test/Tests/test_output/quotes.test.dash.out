   27 ../include/utils.h
   88 ../include/log.h
 1447 ../src/main.c
 1562 total
   27 ../include/utils.h
   88 ../include/log.h
 1447 ../src/main.c
 1562 total
27+88+1447+1562
../include/log.h
../include/utils.h
../src/main.c
*.c
